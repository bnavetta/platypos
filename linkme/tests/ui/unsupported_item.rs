use linkme::distributed_slice;

#[distributed_slice]
pub static SLICE: [&'static str] = [..];

#[distributed_slice(SLICE)]
extern crate std as _std;

fn main() {}
