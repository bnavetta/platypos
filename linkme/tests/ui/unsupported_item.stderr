error: distributed element must be either static or function item
 --> tests/ui/unsupported_item.rs:7:1
  |
7 | extern crate std as _std;
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
